modules/nw-endpoint-route53.adoc

```diff
@@ -24,13 +24,13 @@ platform:
     region: us-gov-west-1
     serviceEndpoints:
     - name: ec2
-       url: https://ec2.us-gov-west-1.amazonaws.com
```

Comment:
Hi @lihongan, can you confirm this YAML update? Thanks!

Reply:
@jboxman It looks good. thank you.